openeo_udp/sentinel1_stats/generate.py

```diff
@@ -0,0 +1,74 @@
+import json
+from pathlib import Path
+
+import openeo
+from openeo.api.process import Parameter
+from openeo.rest.udp import build_process_dict
+from openeo.processes import array_create, array_concat
+
+# TODO #15 where to put reusable helpers? e.g. load description from README.md, dummy openeo connection, properly write to JSON file, ...
+
+
+def generate() -> dict:
+    # TODO: use/inject dummy connection instead of concrete one? (See cdse_marketplace_services)
+    connection = openeo.connect("openeofed.dataspace.copernicus.eu")
+
+    # define parameters
+    spatial_extent = Parameter.bounding_box(
+        name="spatial_extent",
+        default={"west": 8.82, "south": 44.40, "east": 8.92, "north": 44.45},
+    )
+    temporal_extent = Parameter.temporal_interval(
+        name="temporal_extent", default=["2023-05-01", "2023-07-30"]
+    )
+
+    # load collection
+    s1_raw = connection.load_collection(
+        collection_id="SENTINEL1_GRD",
+        temporal_extent=temporal_extent,
+        spatial_extent=spatial_extent,
+        bands=["VH", "VV"],
```

Comment:
bands could be a parameter, with default.

Reply:
indeed, I don't think we can do dynamic band name generation yet, would need to dive into openeo processes to see if it is at all feasible with the current set.
So suggest to leave it to default bands for now.